tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/op/math/Abs.java

```diff
@@ -50,6 +50,7 @@
   @Endpoint(describeByClass = true)
   public static <T extends TNumber> Abs<T> create(Scope scope, Operand<T> x) {
     OperationBuilder opBuilder = scope.env().opBuilder("Abs", scope.makeOpName("Abs"));
+    opBuilder.setDevice(scope.makeDeviceString());
```

Comment:
All generated ops should have been update by your change in the op generator above, they should be all updated by this PR as well, not only this one (there must be 1000+ files to be added).